test/error-message-lister.js

```diff
@@ -0,0 +1,242 @@
+        stepName:          'Step',
+        relatedSourceCode: 'code',
+        action:            'test-action',
+        element:           'element'
+    },
+    {
+        code:              TYPE.incorrectDraggingSecondArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code',
+        action:            'action'
+    },
+    {
+        code:              TYPE.incorrectPressActionArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.emptyTypeActionArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:     TYPE.unexpectedDialog,
+        stepName: 'Step',
+        dialog:   'test-dialog',
+        message:  'message'
+    },
+    {
+        code:     TYPE.expectedDialogDoesntAppear,
+        stepName: 'Step',
+        dialog:   'test-dialog'
+    },
+    {
+        code:              TYPE.incorrectSelectActionArguments,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.incorrectWaitActionMillisecondsArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.incorrectWaitForActionEventArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.incorrectWaitForActionTimeoutArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.waitForActionTimeoutExceeded,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.emptyIFrameArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.iframeArgumentIsNotIFrame,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.multipleIFrameArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.incorrectIFrameArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.uploadCanNotFindFileToUpload,
+        stepName:          'Step',
+        relatedSourceCode: 'code',
+        filePaths:         ['path1', 'path2']
+    },
+    {
+        code:              TYPE.uploadElementIsNotFileInput,
+        stepName:          'Step',
+        relatedSourceCode: 'code',
+        filePath:          'path'
+    },
+    {
+        code:              TYPE.uploadInvalidFilePathArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    }
+];
+
+var testMock = {
+    name:    'fixtureTest',
+    fixture: {
+        name: 'fixture',
+        path: './fixture.js'
+    }
+};
+
+var browserConnectionMocks = [
+    { userAgent: 'Chrome' }
```

Comment:
Maybe it should be named just `browserConnectionMock` and just do `this.browserConnections = [browserConnectionMock]` like it's done for the test? Also it's better to use full UA string (which will be produced by reporter): e.g. `Chrome 15.0.874 / Mac OS X 10.8.1`